scaling_performance/optimizing_storage.adoc

```diff
@@ -14,10 +14,16 @@ toc::[]
 Optimizing storage helps to minimize storage use across all resources. By optimizing storage,
 administrators help ensure that existing storage resources are working in an efficient manner.
 
-This guide primarily focuses on optimizing persistent storage.  Local
-ephemeral storage  _(if the ephemeral-storage tech preview in
-{product-title} 3.10 has been enabled)_, for data utilized during the lifetime of pods, has
-fewer options.
+[NOTE]
+====
+This guide primarily focuses on optimizing persistent storage. Local
```

Comment:
I might say something like
"If you enabled the ephemeral storage technology feature preview in {product-title} 3.10, you can also xref:../install_config/configuring_ephemeral.adoc[configure ephemeral storage], which is temporary storage that pods use during their lifetimes. This feature is disabled by default."